js/tests/e2e/tests/join.spec.js

```diff
@@ -0,0 +1,11 @@
+import { test, expect } from "@playwright/test";
+
+test("join session", async ({ page }) => {
+  await page.goto(`http://127.0.0.1:2999?sessionId=${process.env.SESSION_ID}`);
```

Comment:
```suggestion
  await page.goto(`/?sessionId=${process.env.SESSION_ID}`);
```
We should rely on the base url in playwright config, not to repeat it in each test.